core/amber/src/main/scala/edu/uci/ics/texera/workflow/operators/source/cache/CacheSourceOpDesc.scala

```diff
@@ -18,7 +18,7 @@ class CacheSourceOpDesc(val targetSinkStorageId: OperatorIdentity, opResultStora
   assert(null != targetSinkStorageId)
   assert(null != opResultStorage)
 
-  override def sourceSchema(): Schema = opResultStorage.get(targetSinkStorageId).getSchema
+  override def sourceSchema(): Schema = opResultStorage.getSchema(targetSinkStorageId)
```

Comment:
better to keep it as the old way, treat the opResultStorage as a map of all storage/document objects. each storage/document object knows its schema.